### DISCUSSION TITLE
Support MYSTROM_BUTTON_PLUS

### DISCUSSION THREAD

**Discussion by daniestermann** [johannrichard/homebridge-dingz#447] [General]
MYSTROM_BUTTON_PLUS is currently unsupported.

Thanks for this plugin!
Any chance to add support for the Button Plus just with the basic single-tap function?
I do not need any special features beyond that.

**Comment by johannrichard** [comment]
Sorry, am a bit slow (very slow) to respond. I don't have a Button Plus at my disposal and would have to have a look at the API first.

**Reply by johannrichard** [reply]
Working on it, progress tracked in #540

**Reply by johannrichard** [reply]
Support is now released w/ v3.3.0.

**Comment by Admiral-slow** [comment]
Any chance that we see wifi button plus 1gen support soon? thanks for your work.
